juju/juju
# Adds support for interactive hidden file creds.

When Juju users have to input interactive credentials for hidden file
based attributes they previously had no way to enter this data. This
change now allows file names to be entered with the user input echoed.

## Checklist


## QA steps

- Create a new lxd cloud with `juju add-cloud`
- Add credentials for the new cloud with `juju add-credentials` choosing the certificate input method during an interactive session. Make sure that filename can be entered for the certificate data and what the user types can be seen.

## Documentation changes

N/A

## Bug reference

https://bugs.launchpad.net/juju/+bug/1988239
